core-web/libs/template-builder/src/lib/directives/container-options.directive.stories.ts

```diff
@@ -0,0 +1,43 @@
+import { moduleMetadata, Story, Meta } from '@storybook/angular';
+
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+
+import { DropdownModule } from 'primeng/dropdown';
+
+import { DotContainersService, DotMessageService } from '@dotcms/data-access';
+import { DotContainersServiceMock } from '@dotcms/utils-testing';
+
+import { ContainerOptionsDirective } from './container-options.directive';
+
+import {
+    DOT_MESSAGE_SERVICE_TB_MOCK,
+    MockContainersDropdownComponent
+} from '../components/template-builder/utils/mocks';
+
+export default {
+    title: 'Container Options Directive',
```

Comment:
Check the title in the other stories, the `/` allows us to put things into different pages. https://storybook.js.org/docs/react/writing-stories/naming-components-and-hierarchy